### ISSUE TITLE
Remove last "return" statement

### ISSUE THREAD

**Issue by AkiraLaine** [FreeCodeCamp/FreeCodeCamp#5628]
Waypoint: Chaining If Else Statements

My code is all correct but it didn't want to pass. The error was "You should have five return statements" - so I removed the "change me" return statement and it passed. 
But the instructions don't tell you to remove the last return statement.

My code:
```javascript
function myTest(num) {
  // Only change code below this line
  if(num < 5){
    return "Tiny";
  } else if (num < 10){
    return "Small";
  } else if (num < 15){
    return "Medium";
  } else if(num < 20){
    return "Large";
  } else if(num >= 20){
    return "Huge";
  }
  return "Change Me"; //We need to remove this to pass
  // Only change code above this line
}

// Change this value to test
myTest(7);

```

**Comment by DDaems** [comment]
Is there any reason why we are expliclity testing for 5 return statements? If not, i think we should remove it.

**Comment by ltegman** [comment]
I'd agree with @DDaems that the return test is not necessary. If the code returns the correct answer and passes the if/else number tests that test isn't important.

**Comment by SaintPeter** [comment]
I would change it to `> 5` rather than `=== 5`

**Comment by pmbenjamin** [comment]
@SaintPeter then anyone who would have exactly five would not pass either. 

To remain as accurate as possible, I would change it to `>= 5`. Otherwise, if you want to keep it as `>5`, then we need to change the description to say "You should have **more than five** return statements."

**Comment by smrajagopal** [comment]
@pmbenjamin : You are right my fix 63f324c147a08ccfbc69e5f8b5647dfcaecfcd65 actually has `>=5`

**Comment by pmbenjamin** [comment]
@smrajagopal, good enough for me!
